### PR TITLE
Added a page for partners of Lernwerkstatt

### PR DESCRIPTION
**Pull request by hikes101**
Added a page of partners including logos.

### CODE REVIEW

**Comment by hikes101** [comment]
[Create Partners page](https://trello.com/c/glg3s3yX/96-create-partners-page)

File: src/routes/index.js [hunk 1]
```diff
@@ -7,17 +7,19 @@ const aboutRouter = require("./about.js");
 const blogsRouter = require("./blogs.js");
 const contactRouter = require("./contact.js");
 const learnRouter = require("./learn.js");
-const supportRouter = require("./support.js");
+const partners = require("./partners.js");
 const projects = require("./projects.js");
 const subscribeRouter = require("./subscribe.js");
+const supportRouter = require("./support.js");
```
Comment (cappyccino) [inline src/routes/index.js]: alphabetized the list

Comment (cappyccino) [inline src/routes/index.js]: alphabetized the list

**Comment by hikes101** [comment]
@pheeria I resolved the merge conflicts. Sorry for the high number of commits, next time we'll squash commits when working together on one branch.

**Pull request merged**